common/src/main/java/com/microsoft/identity/common/internal/fido/WebAuthnJsonUtil.kt

```diff
@@ -87,8 +93,16 @@ class WebAuthnJsonUtil {
                 WEBAUTHN_RESPONSE_CLIENT_DATA_JSON_KEY))
             assertionResult.put(WEBAUTHN_RESPONSE_SIGNATURE_JSON_KEY, authResponseJsonObject.get(
                 WEBAUTHN_RESPONSE_SIGNATURE_JSON_KEY))
-            assertionResult.put(WEBAUTHN_RESPONSE_USER_HANDLE_JSON_KEY, authResponseJsonObject.get(
-                WEBAUTHN_RESPONSE_USER_HANDLE_JSON_KEY))
+            // UserHandle is optional if allowCredentials was provided in the request (username flow).
```

Comment:
should we also log if allowCeredentials was provided.

Reply:
Makes sense: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/2560/commits/a58e216fca74dcb351c4297516ff9b4aae5d5d2c